hpOneView/test/unit/test_suite.py

```diff
@@ -1,41 +0,0 @@
-# -*- coding: utf-8 -*-
-###
-# (C) Copyright (2012-2016) Hewlett Packard Enterprise Development LP
-#
-# Permission is hereby granted, free of charge, to any person obtaining a copy
-# of this software and associated documentation files (the "Software"), to deal
-# in the Software without restriction, including without limitation the rights
-# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-# copies of the Software, and to permit persons to whom the Software is
-# furnished to do so, subject to the following conditions:
-#
-# The above copyright notice and this permission notice shall be included in
-# all copies or substantial portions of the Software.
-#
-# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-# THE SOFTWARE.
-###
-import unittest
-from test_profile import ProfilesTest
-from test_networking import NetworkingTest
-from test_servers import ServersTest
-
-
-class TestSuite (unittest.TestCase):
-
-    def suite():
-        suite = unittest.TestSuite()
-        suite.addTest(ProfileTest())
-        suite.addTest(NetworkingTest())
-        suite.addTest(ServersTest())
-        return suite
-
-if __name__ == '__main__':
-    runner = unittest.TextTestRunner()
-    test_suite = suite()
-    runner.run(test_suite)
```

Comment:
For running the tests, we suggest using tox as test suit
The reasons we prefer tox are:
•	Tox will automatically search all created tests
•	The suite class requires maintenance (adding every new test)
•	Depending of the way of execution, for example: python -m unittest discover, the tests ran more than once, so we removed the test_suite.py
•	Project should support Python 2.7+ and Python 3.4+. Tox automates the test execution in each Python version

Below are the commands to install an run tox (the tox script will come in the next pull request)

`$ pip install tox`
enter the project folder
`$ tox`